src/scrapers/main.py

```diff
@@ -1,16 +1,19 @@
 import logging
 from optparse import OptionParser
```

Comment:
Deprecated since version 3.2: The optparse module is deprecated and will not be developed further; development will continue with the argparse module.

Reply:
`from argparse import ArgumentParser`